rsocket-core/src/main/java/io/rsocket/lease/LeaseRSocket.java

```diff
@@ -56,7 +56,7 @@ public double availability() {
         : 0.0;
```

Comment:
Can lease.getStartingAllowedRequests() every equal zero? Might want to check for that.

Reply:
lease.startingNumberOfRequests is set in constructor and never changes, so only case when it is 0 is when lease.numberOfRequests is 0, but then lease.isValid is false